packages/mgt-components/src/components/sub-components/mgt-dot-options/mgt-dot-options.ts

```diff
@@ -79,11 +79,11 @@ export class MgtDotOptions extends MgtBaseComponent {
     const menuOptions = Object.keys(this.options);
 
     return html`
-      <span
-        tabindex="0"
+      <fluent-button
+        appearance="stealth"
         @click=${this.onDotClick}
         @keydown=${this.onDotKeydown}
-        class="DotIcon">\uE712</span>
+        class="DotIcon">\uE712</fluent-button>
```

Comment:
Should we use an SVG here?

Reply:
🤔 I'm not sure if it makes a difference. Maybe a consistency issue where we do away with using the `FabricMDL2Icons` font-family usage. What do you think?

Reply:
(marking as resolved to allow the PR to complete)